app/models/pydantic/creation_options.py

```diff
@@ -339,6 +340,26 @@ class RasterTileCacheCreationOptions(TileCacheBaseModel):
     )
 
 
+class COGCreationOptions(StrictBaseModel):
+    implementation: str = Field(
+        "default",
+        description="Name space to use for COG. "
+        "This will be part of the URI and will "
+        "allow to create multiple COGs per version,",
```

Comment:
Optional: Change "... per version,"  to "... per version" (get rid of extra comma).  Also, maybe change:

"allow to create multiple COGs per version"

to

"allow creation of multiple COGs per version".